src/async_jobs.ts

```diff
@@ -414,6 +501,22 @@ async function slackInteractivityHandler(
         return;
       }
 
+      case SlackCallbackId.ROUTE_TO_JOURNEY: {
+        const modalPrivateMetadata = JSON.parse(
```

Comment:
I'd recommend explicitly checking the type of this data rather than assuming that it will be valid JSON -- maybe something like `parseSlackModalPrivateMetadata(metadata: string): SlackModalPrivateMetadata`. So that we get better errors if we accidentally open a modal with invalid data, or if the expected data changes in between versions of the helpline and someone opens a modal, then we deploy new code, and then they click a button in the modal.

Reply:
Done. I hope this is what you had in mind.